rasa/cli/train.py

```diff
@@ -81,8 +75,10 @@ def run_training(args: argparse.Namespace, can_exit: bool = False) -> Optional[T
     domain = rasa.cli.utils.get_validated_path(
         args.domain, "domain", DEFAULT_DOMAIN_PATH, none_is_valid=True
     )
+    config = rasa.cli.utils.validate_config_path(args.config)
```

Comment:
These two lines are repeated a lot in the code. We should consider to move them to a function or a method.

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11978/commits/1f874275f22df4119245d1d1326de818bf041adb